.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 package-lock.json
+.idea/*
```

Comment:
should be `.idea` without the `/*` so that the `.idea/workspace.xml` will also be ignored